Source/SwiftLintFramework/Rules/QuickDiscouragedCallRule.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  QuickDiscouragedCallRule.swift
+//  SwiftLint
+//
+//  Created by Ornithologist Coder on 8/11/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+// swiftlint:disable identifier_name
+
+public struct QuickDiscouragedCallRule: ASTRule, OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "quick_discouraged_call",
+        name: "Quick Discouraged Call",
+        description: "Discouraged call inside 'describe' and/or 'context' block.",
+        kind: .style,
```

Comment:
I think this could be `.lint`, since it's more related to semantics and potential issues than to code style.